src/redux/actions/samples/updateSampleFile.js

```diff
@@ -16,18 +17,22 @@ const updateSampleFile = (
 ) => async (dispatch, getState) => {
   const updatedAt = moment().toISOString();
   const sample = getState().samples[sampleUuid];
+  // we'll need to remove the hard-coded 10x tech type once we start
+  // supporting other types and save the chosen tech type in redux
+  const valid = checkIfFileValid(fileName, '10X Chromium');
 
   try {
     // Save sample only if upload is successful or error
     if (fileDiff.upload.status === UploadStatus.UPLOADED
```

Comment:
this is not part fo your PR, but could you perhaps change this if for `[UPLOADED, UPLOAD_ERROR].includes(fileDiff.upload.status)`